teamcity/README.md

```diff
@@ -14,7 +14,7 @@ The TeamCity check is included in the [Datadog Agent][1] package, so you don't n
 
 #### Prepare TeamCity
 
-1. To prepare TeamCity, see [Enabling Guest Login][2].
+1. [Enable guest login][2] or identify user credentials (with required permissions) for basic HTTP authentication.
```

Comment:
```suggestion
You can enable [Guest login](#guest-login), or identify [user credentials](#user-credentials) for basic HTTP authentication.

##### Guest login

1. [Enable guest login][2] or identify user credentials (with required permissions) for basic HTTP authentication.
```

Reply:
Thank you @buraizu , accepted the change.